### PR TITLE
Create de.yml

### PR DESCRIPTION
**Pull request by Idrinth**
# The Pull Request is ready

- [ ] all actions are passing
- [ ] only fixes a single issue

## Overview

German transslation

## Translations

- [ ] the translation is appropriate for all audiences
- [ ] the translation is not an unchecked automatic translation

### CODE REVIEW

File: documentation-website/language/de.yml [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+  slack:
+    title: "@idrinth/api-bench on slack"
+    alt: "Slack"
+  linkedin:
+    title: "@idrinth/api-bench on LinkedIN"
+    alt: "LinkedIn"
+  open-sauced:
+    title: "@idrinth/api-bench projects on OpenSauced"
+    alt: "OpenSauced"
+
```
Comment (BrinaBre) [inline documentation-website/language/de.yml:57]: Ab hier weiter machen

File: documentation-website/language/de.yml [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+code-of-conduct:
+  meta:
+    title: "Verhaltenskodex"
+    description: "Der Verhaltenskodex für Beitragende des @idrinth/api-bench"
+
+home:
+  meta:
+    title: "Startseite"
+    description: "Willkommen in der Dokumentation für das Framework @idrinth/api-bench"
```
Comment (BrinaBre) [inline documentation-website/language/de.yml:66]: weiter

**Comment by Idrinth** [comment]
@coderabbitai review

File: documentation-website/language/de.yml [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+    title: "Zu bestehenden Bibliotheken"
+    description: "Andere Tools wie matteofigus/api-benchmark, bvanderlaan/api-bench-runner oder jeffbski/bench-rest wurden seit geraumer Zeit nicht mehr aktualisiert und bieten keine Typdefinitionen für TypeScript. Dies macht sie weniger attraktiv für Projekte, in denen eine bessere statische Codeprüfung die Entwicklungsgeschwindigkeit erheblich erhöht."
+    reusability: "Ein großer Vorteil der Verwendung dieses Frameworks besteht darin, dass Routendefinitionen einmal geschrieben und für alle gewünschten Testtypen wiederverwendet werden können."
+    threading: "Darüber hinaus trennt dieses Tool den Validierungsthread vom Thread, der die tatsächlichen Anfragen verarbeitet, um den Effekt komplizierter Validierungen oder großer Antwortdatenmengen auf die Datenerfassung weiter zu minimieren."
+
+faq:
+  meta:
+    title: "FAQ"
+    description: "Häufig gestellte Fragen zu @idrinth/api-bench"
+  title: "Häufig gestellte Fragen"
```
Comment (BrinaBre) [inline documentation-website/language/de.yml:92]: Fertig

File: documentation-website/language/de.yml [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+    description_2: "Lasttests bewerten das Verhalten eines Systems unter Spitzen- oder erwarteten Lastbedingungen."
+    title_3: "Wie lange dauert es einen Test zu schreiben?"
+    description_3: " Die durschnittliche Zeit beträgt 2-3 Minuten.Es kann aber bis zu 15 Minuten dauern. "
+    title_4: "Wie lange dauert die Einrichtung?"
+    description_4: "Wenn Sie dem Schnellstart folgen, dauert es nur wenige Minuten!"
+    title_5: "Ist das Projekt produktiv im Einsatz?"
+    description_5: "Ja, Das Projekt ist sehr produktiv im Einsatz."
+    title_6: "Gibt es zusätzliches Unterrichtsmaterial?"
+    description_6: "Noch nicht, es wird aktiv daran gearbeitet."
+    title_7: "Ist die Nutzung des Frameworks kostenlos?"
```
Comment (BrinaBre) [inline documentation-website/language/de.yml:107]: Fertig

**Pull request closed without merging**